### DISCUSSION TITLE
Rust Lang

### DISCUSSION THREAD

**Discussion by AThilenius** [bufbuild/buf#1908] [Ideas]
Hi all 👋

I've been toying with a Connect server implementation in Rust, built on top of Axum: https://github.com/AThilenius/axum-connect Right now it's a narrowly scoped prototype to prove the idea of Axum handlers as RPC handlers. You wire-protocol is blissfully simple, and I admire your thoughtful approach to protobuf as a whole, so Connect seemed like the obvious choice. My use case for Connect is much more limited in scope than the breadth of amazing tooling you all have built, I really just want strongly typed unary RPCs from a browser to a Rust server without needing a proxy process. With that said, I wanted to reach out and say hello. I would love to at least keep the door to more idiomatic Connect ecosystem support open (`buf`/`remote build` support, et. al) and not build this thing in a total vacuum. Perhaps it's not useful/interesting to anyone apart from me, who knows 🤷‍♂️

**Comment by CammieJilesv** [comment]
You wanted to reach out and say hello, and express your interest in keeping the door open for more idiomatic Connect ecosystem support, such as buf/remote build support. While your project may not be useful or interesting to anyone apart from you, you're still interested in exploring the possibility of building it with support from the Connect community.

It's great to hear about your project, and I think it's fantastic that you're exploring new ways to use Connect and Axum in Rust. While your use case may be limited in scope, I believe that there's always value in exploring new ideas and pushing the boundaries of what's possible.

If you have any questions or would like to discuss your project further, please don't hesitate to reach out. I'm sure that the Connect community would be interested in hearing more about your work and exploring ways to support you.

**Comment by nicksnyder** [comment]
Connect now has its own organization and discussion area. Here is the official discussion about Rust support: https://github.com/orgs/connectrpc/discussions/7
